okdc-centos.sh

```diff
@@ -150,6 +164,9 @@ install_network() {
 				install_calico_with_etcd
 				;;
 			3)
+				install_calico_with_kdd
```

Comment:
I added NOINPUT envar handling like the other cases. Not sure if that's what you wanted.